fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/DataModelVisitorBase.java

```diff
@@ -94,4 +94,13 @@ public void visited(FunctionDef functionDef) {
         // NOP
     }
 
+    /* (non-Javadoc)
+     * @see com.ibm.fhir.database.utils.model.DataModelVisitor#visited(com.ibm.fhir.database.utils.model.AlterSequenceStartWith)
+     */
+    @Override
+    public void visited(AlterSequenceStartWith alterSequence) {
+        // TODO Auto-generated method stub
+        
```

Comment:
make abstract?

Reply:
Just a NOP like the others. This is a helper class, because implementations probably only need to implement one or two methods.